authorization-service/serverless.ts

```diff
@@ -0,0 +1,75 @@
+const serverlessConfiguration = {
+    service: 'authorization-service',
+    useDotenv: true,
+    provider: {
+      name: 'aws',
+      runtime: 'nodejs20.x',
+      stage: 'dev',
+      region: 'eu-west-1',
+      apiGateway: {
+        minimumCompressionSize: 1024,
+        shouldStartNameWithService: true,
+      },
+      environment: {
+        REGION: 'eu-west-1',
```

Comment:
You can reference region property in provider section in order not to copy paste (if you change one place, you can forgot to change other place some day):

https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml

Reply:
done